.clang-tidy

```
@@ -0,0 +1,2 @@
+Checks: 'bugprone-*,performance-*,modernize-*,readability-*,cppcoreguidelines-*'
```

Highly recommended we do one-per-line. Much easier to read visually and for Git to diff when we start tweaking checks.

```suggestion
Checks: >
  bugpront-*,
  performance-*,
  modernize-*,
  readability-*,
  cppcoreguidelines-*
```

(need to verify the above syntax -- taken blindly from StackOverflow https://stackoverflow.com/a/60723032